src/utilities/indexer/updateAttestationCount.ts

```diff
@@ -0,0 +1,52 @@
+import { type ICType } from '@kiltprotocol/sdk-js';
+
+import { Op } from 'sequelize';
+
+import { CType as CTypeModel } from '../../models/ctype';
+
+import { logger } from '../logger';
+
+import { matchesGenerator } from './queryFromIndexer';
+
+export async function updateAttestationsCount() {
+  const fields = ['cTypeId: id', 'attestationsCreated', 'registrationBlockId'];
+
+  const queryParams = {
+    entity: 'cTypes',
+    alias: 'attestationsCount',
+    fields,
+  };
+
+  interface QueriedAttestationCount {
+    cTypeId: ICType['$id'];
+    attestationsCreated: number;
+    registrationBlockId: string; // Block Ordinal Number, without punctuation
+  }
```

Comment:
I believe there’s a general agreement that interfaces should be put at the top level of modules, with a rare exception of them depending on parameter types

Reply:
I move all my interfaces outside of the functions.

Reply:
Should `fieldsToQuery` and `queryParams` also be outside of the function? They correlate with the interfaces.

Reply:
Yes, I guess so, but I also believe they will not be needed when the query is generated in-place

Reply:
> Yes, I guess so, but I also believe they will not be needed when the query is generated in-place

if**

Reply:
Can't take out the `queryParams` out of the function because of this line:
```
    filter: `{ registrationBlock: { timeStamp: { greaterThan: "${fromDate.toISOString()}" } } }`,
```

Reply:
Moved it outside as well.